configurations/morcore2esuites/Configuration_ESBZGWApi.xml

```diff
@@ -79,7 +124,9 @@
                 <Param name="opdrachtInfo" sessionKey="opdrachtInfo" />
             </XsltPipe>
 
-            <JsonPipe name="XmlToJson" direction="XML2JSON" />
+            <JsonPipe name="XmlToJson" direction="XML2JSON">
+                <Forward name="success" path="SendToESB"/>
+                <Forward name="exception" path="exception"/>
 
             <SenderPipe name="SendToESB">
                 <HttpSender name="SendToESB" methodType="POST" url="/melding/{meldingUuid}"
```

Comment:
The same need to be done for communication with the ESB as with MorCore. Url not as attribute, but as param. And need to start with the root-url for the ESB. So for the ESB root-url you would also add a property to DeploymentSpecifics like: "esuites-esb.root-url=" for example. and then reference it in your url. 

Also {meldingUuid} -> ${meldingUuid}. If meldingUuid is not a sessionKey, it need to be added as a param to provide the value for it.

<SenderPipe name="SendToESB">
                <HttpSender name="SendToESB" methodType="POST"
                    headersParams="X-Audit-Toelichting,Accept-Crs,Content-Crs">
                    <Param name="url" value="${esuites-esb.root-url}/melding/${meldingUuid}" />
                    <Param name="X-Audit-Toelichting" sessionKey="toelichtingHeader" />
                    <Param name="Accept-Crs" value="EPSG:4326" />
                    <Param name="Content-Crs" value="EPSG:4326" />
                </HttpSender>